### PR TITLE
Fix license + css file name

### PR DESCRIPTION
**Pull request by camelburrito**
Headings for licenses.

Write unminified css file as max.css (instead of amp.css) in the zip file

### CODE REVIEW

**Comment by camelburrito** [comment]
@ericlindley-g - we might have to add some more wordings on `How it works` since we now have unminified CSS

**Comment by ericlindley-g** [comment]
Gotcha — what's changed about what they need to do? Do they now always need to minify after modifying, or are we providing minified as well? And do the files all fit under the 50K limit anyway?

File: tasks/bundle.js [hunk 1]
```diff
@@ -110,7 +113,8 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
-
+Normalize | https://github.com/necolas/normalize.css/blob/master/LICENSE.md
```
Comment (tonyruscoe) [inline tasks/bundle.js]: Nit: I think the official project name is called “Normalize.css”

File: tasks/bundle.js [hunk 2]
```diff
@@ -121,6 +125,8 @@ The above copyright notice and this permission notice shall be included in all c
 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 
+AMP Start | https://github.com/ampproject/ampstart/blob/master/LICENSE
```
Comment (tonyruscoe) [inline tasks/bundle.js]: Is it “AMP Start” or “AMPStart” (as in the license below)? Should really be consistent.

**Pull request merged**